src/build.ts

```diff
@@ -171,6 +164,12 @@ export async function writeTypes(nitro: Nitro) {
       resolvedImportPathMap.set(i.from, path);
     }
 
+    // TODO: fully resolve utils exported from `#imports`
+    autoImportExports = await nitro.unimport
+      .toExports(typesDir)
+      .then((r) =>
+        r.replace(/'(.*)'/g, (_, i) => `'${resolvedImportPathMap.get(i) ?? i}'`)
```

Comment:
This might be also something potentially unimport could directly solve. /cc @antfu